src/offset/local/unix.rs

```diff
@@ -31,69 +31,70 @@ thread_local! {
     static TZ_INFO: RefCell<Option<Cache>> = Default::default();
 }
 
+// weird looking function signature
+// but allows sharing the results of env::var around
+fn get_tz_env_hash(env_result: &Result<String, env::VarError>) -> Option<u64> {
```

Comment:
Given that this is single-caller, not sure why extracting it as a separate function is useful.

Reply:
removed